ksql-rest-model/src/main/java/io/confluent/ksql/rest/entity/KsqlStatementErrorMessage.java

```diff
@@ -77,12 +76,12 @@ public boolean equals(final Object o) {
       return false;
     }
     final KsqlStatementErrorMessage that = (KsqlStatementErrorMessage) o;
-    return Objects.equals(statementText, that.statementText)
-           && Objects.equals(entities, that.entities);
+    return Objects.equals(entities, that.entities);
   }
 
   @Override
   public int hashCode() {
-    return Objects.hash(super.hashCode(), statementText, entities);
+    return Objects.hash(super.hashCode(),
+        io.confluent.ksql.rest.entity.KsqlStatementErrorMessage.statementText, entities);
```

Comment:
nit: remove the statementText from this hash function, since it's constant anyway.